src/client.js

```diff
@@ -56,7 +57,11 @@ export default class StitchClient {
       }
     };
 
-    this.auth = new Auth(this, this.authUrl);
+    const authOptions = {codec: APP_CLIENT_CODEC};
+    if (options && options.authCodec) {
+      authOptions.codec = options.authCodec;
+    }
+    this.auth = new Auth(this, this.authUrl, authOptions);
```

Comment:
I think `authUrl` should be in the constructor inside of an `authConfig` object as `url`. Same for options as `options`. we should try to make `StitchClient` not have logic to for admin functionality and let `Admin` handle it.